src/components/include/security_manager/crypto_manager.h

```diff
@@ -71,6 +71,7 @@ class CryptoManager : public policy::PolicyHandlerObserver {
   * \return pointer to crypto manager settings class
   */
   virtual const CryptoManagerSettings& get_settings() const = 0;
+  virtual void SetCertExpTime(const std::string& time) = 0;
```

Comment:
@LuxoftAKutsan It was added by this PR: https://github.com/smartdevicelink/sdl_core/pull/1434
Do you suggest - I have to remove all changes provided by this PR?